### PR TITLE
Manual and modified script update

### PR DESCRIPTION
**Pull request by manjunath-grl**
Modified tests
TC-APPLAUNCHER-1.3
TC-APPLAUNCHER-3.5
TC-APPLAUNCHER-3.6
TC-BOOL-2.1
TC-CC-3.3
TC-CHANNEL-1.6
TC-CHANNEL-5.1
TC-CHANNEL-5.2
TC-CHANNEL-5.3
TC-I-1.1
TC-LVL-2.1
TC-LVL-2.2
TC-LVL-3.1
TC-LVL-4.1
TC-LVL-5.1
TC-LVL-6.1
TC-MEDIAINPUT-1.4
TC-MEDIAINPUT-3.10
TC-MEDIAINPUT-3.11
TC-MEDIAINPUT-3.12
TC-MEDIAINPUT-3.13
TC-CADMIN-1.3
TC-CADMIN-1.4
TC-CADMIN-1.5
TC-CADMIN-1.6
TC-CADMIN-1.9
TC-CADMIN-1.10
TC-CADMIN-1.15
TC-OO-2.3
Modifed Manual scripts
TC-CADMIN-1.*
TC-MOD-1.*
Added auto generated files

Attached execution logs:
[Execution_log.txt](https://github.com/project-chip/connectedhomeip/files/9123308/Execution_log.txt)

### CODE REVIEW

File: src/app/tests/suites/certification/Test_TC_LVL_2_2.yaml [hunk 1]
```diff
@@ -52,6 +52,7 @@ tests:
           value: 10
           constraints:
               type: uint16
+              notValue: OnOffTransitionTimeValue
```
Comment (bzbarsky-apple) [inline src/app/tests/suites/certification/Test_TC_LVL_2_2.yaml:55]: Why not?  What if the original value was 10?

File: src/app/tests/suites/certification/Test_TC_LVL_2_2.yaml [hunk 2]
```diff
@@ -67,14 +68,14 @@ tests:
...
     - label: "Reads the OnLevel attribute from the DUT"
       PICS: LVL.S.A0011
       command: "readAttribute"
       attribute: "on level"
       response:
-          value: 254
+          value: 5
           constraints:
               type: uint8
               notValue: OnLevelValue
```
Comment (bzbarsky-apple) [inline src/app/tests/suites/certification/Test_TC_LVL_2_2.yaml:81]: What if the original value was 5?

File: src/app/tests/suites/certification/Test_TC_LVL_2_2.yaml (hunk 1 above)
Comment (manjunath-grl) [reply src/app/tests/suites/certification/Test_TC_LVL_2_2.yaml:55]: @bzbarsky-apple i will update the script with configurable values

File: src/app/tests/suites/certification/Test_TC_LVL_2_2.yaml (hunk 2 above)
Comment (manjunath-grl) [reply src/app/tests/suites/certification/Test_TC_LVL_2_2.yaml:81]: @bzbarsky-apple i will update the script with configurable values

File: src/app/tests/suites/certification/Test_TC_LVL_2_2.yaml (hunk 1 above)
Comment (manjunath-grl) [reply src/app/tests/suites/certification/Test_TC_LVL_2_2.yaml:73]: @bzbarsky-apple[ TC-LVL-2.2](https://github.com/project-chip/connectedhomeip/blob/53415c2ad4cfa98f51128209b5060427ac2dbc71/src/app/tests/suites/certification/Test_TC_LVL_2_2.yaml) is modified and rebased can you please re-verify it

**Review by woody-apple — APPROVED** [review]

**Pull request merged**